sqldelight-compiler/src/test/kotlin/app/cash/sqldelight/core/QueriesTypeTest.kt

```diff
@@ -10,9 +10,11 @@ import org.junit.Test
 import org.junit.rules.TemporaryFolder
 
 class QueriesTypeTest {
-  @get:Rule val temporaryFolder = TemporaryFolder()
+  @get:Rule
+  val temporaryFolder = TemporaryFolder()
 
-  @Test fun `queries file is generated properly via compilation`() {
+  @Test
+  fun `queries file is generated properly via compilation`() {
```

Comment:
Do you use a specific formatting template? I did Intellij format code and got this change. @griffio

Reply:
Done, it passes.

Reply:
no - I don't use Intellij to format code in this project - just `./gradlew :spotlessApply` before commits

Reply:
QueriesTypeTest.kt - I would suggest removing the formatting changes to the other tests and keep to the existing style. 

It will be one of the maintainers that approve the changes

Reply:
Done